framework/helpers/BaseInflector.php

```diff
@@ -231,13 +231,46 @@ class BaseInflector
         'ø' => 'o', 'ù' => 'u', 'ú' => 'u', 'û' => 'u', 'ü' => 'u', 'ű' => 'u', 'ý' => 'y', 'þ' => 'th',
         'ÿ' => 'y',
     ];
+
     /**
-     * @var mixed Either a [[Transliterator]] or a string from which a [[Transliterator]]
-     * can be built for transliteration used by [[slug()]] when intl is available.
+     * @var mixed Either a [[Transliterator]], string name of preset defined in [[transliteratorAliases]]
+     * or a string from which a [[Transliterator]] can be built for transliteration.
+     * Used by [[transliterate()]] when intl is available.
      * @see http://php.net/manual/en/transliterator.transliterate.php
      */
-    public static $transliterator = 'Any-Latin; NFKD';
+    public static $transliterator = 'loose';
 
+    /**
+     * @var array The array of transliterator aliases.
+     *  - key: alias name
+     *  - value: either a [[Transliterator]] or a string from which a [[Transliterator]] can be built.
+     *
+     * Used by [[transliterate()]] when intl is available.
+     *
+     * The following aliases are defined by default:
+     *  - `strict`: `Any-Latin; NFKD`. Transliteration rules are strict, letters will be transliterated with
+     * the closest sound-representation chars. The result my contain any UTF-8 chars. For example:
+     * `获取到 どちら Українська: ґ,є, Српска: ђ, њ, џ! ¿Español?` will be transliterated to
+     * `huò qǔ dào dochira Ukraí̈nsʹka: g̀,ê, Srpska: đ, n̂, d̂! ¿Español?`
+     * - `medium`: `Any-Latin; Latin-ASCII`. Transliteration rules are medium-strict, letters will be transliterated
+     * to characters of Latin-1 ASCII table. The result still my contain some special chars. For example:
+     * `获取到 どちら Українська: ґ,є, Српска: ђ, њ, џ! ¿Español?` will be transliterated to
+     * `huo qu dao dochira Ukrainsʹka: g,e, Srpska: d, n, d! ¿Espanol?`
+     * - `loose`: `Any-Latin; Latin-ASCII; [\u0080-\uffff] remove`. Transliteration rules are loose, letters will be
+     * transliterated to characters of ASCII table. For example:
+     * `获取到 どちら Українська: ґ,є, Српска: ђ, њ, џ! ¿Español?` will be transliterated to
+     * `huo qu dao dochira Ukrainska: g,e, Srpska: d, n, d! Espanol?`
+     *
+     * You can set your own aliases.
+     * For detailed information see [unicode normalization forms](http://unicode.org/reports/tr15/#Normalization_Forms_Table)
+     * @see http://unicode.org/reports/tr15/#Normalization_Forms_Table
+     * @see transliterate()
+     */
+    public static $transliteratorAliases = [
```

Comment:
Instead of array constants could be used:

```php
const TRANSLITERATOR_STRICT = 'Any-Latin; NFKD';
const TRANSLITERATOR_MEDIUM = 'Any-Latin; Latin-ASCII';
const TRANSLITERATOR_LOOSE = 'Any-Latin; Latin-ASCII; [\u0080-\uffff] remove';
```